app/bundles/QueueBundle/Form/Type/ConfigType.php

```diff
@@ -46,13 +47,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $protocolChoices = array_merge(['' => 'mautic.queue.config.protocol.disabled'], $event->getProtocolChoices());
         $builder->add(
             'queue_protocol',
-            'choice',
+            ChoiceType::class,
             [
-                'label'      => 'mautic.queue.config.protocol',
-                'label_attr' => ['class' => 'control-label'],
-                'data'       => $options['data']['queue_protocol'],
-                'choices'    => $protocolChoices,
-                'attr'       => [
+                'label'               => 'mautic.queue.config.protocol',
+                'label_attr'          => ['class' => 'control-label'],
+                'data'                => $options['data']['queue_protocol'],
+                'choices'             => array_flip($protocolChoices),
+                '‘choices_as_values’' => true,
```

Comment:
```suggestion
                'choices_as_values' => true,
```
Please try to test the Mautic global configuration form. If it loads and if you can save values tot the Queue Settings form.

Reply:
@mtshaw3 ping